app/views/documents/index.js.erb

```diff
@@ -0,0 +1,42 @@
+$("tbody").replaceWith("<tbody><%= j render partial: "documents/document", collection: @documents.select {|document|
```

Comment:
1 argument per line if you make multiline method calls please.
You could also add new line after the first HTML tag, and before the last one, if that makes it easier to read

Reply:
Hi Tim, I followed your advice on the comment above, but I can't in this case as jQuery is white space sensitive.